vscode-lean4/src/utils/fsExistHelper.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { promises } from 'fs';
+
+
+export async function fsExistHelper(pathFile: string): Promise<boolean> {
```

Comment:
It looks like this function doesn't use its argument. What is `this._temporaryStorage`?